src/components/DefaultRedirectionSnackbar/helpers.js

```diff
@@ -1,102 +1,43 @@
-import { deconstructRedirectLink, hasQueryBeenLoaded } from 'cozy-client'
-import { isFlagshipApp } from 'cozy-device-helper'
+import {
+  deconstructRedirectLink,
+  hasQueryBeenLoaded,
+  useSetting
+} from 'cozy-client'
 
 export const VIEW_COUNT_THRESHOLD = 3
 
 export const HOME_DEFAULT_REDIRECTION = 'home/'
 
-export const incrementDefaultRedirectionViewCount = async (
-  client,
-  homeSettings
-) => {
-  const { default_redirection_view_count } = homeSettings
-  const newHomeSettings = {
-    _type: 'io.cozy.home.settings',
-    ...homeSettings,
-    default_redirection_view_count: default_redirection_view_count
-      ? default_redirection_view_count + 1
-      : 1
```

Comment:
I do not see anymore this logic of setting 1 if there was no value before. Will it work ?

Reply:
Fixed [here](https://github.com/cozy/cozy-home/compare/f17fe1ae5ddae709aec71980566f2dc9658dae08..7070fa5d79c09e42c6a980af74c0322d19930999) and in [cozy-client](https://github.com/cozy/cozy-client/pull/1460/commits/98cfaddd2f1e10ceb0d4259962aa4f473de95154)

Reply:
Finally we did not include the default parameter option in the useSettings API, so I added the necessary code in this cozy-home PR